Makefile

```diff
@@ -93,7 +93,7 @@ endif
 		cp -R plugins/$(PLUGIN_NAME) ${CORE_PLUGINS_PATH}/available && \
 		rm -rf ${CORE_PLUGINS_PATH}/available/$(PLUGIN_NAME)/src && \
 		ln -s ${CORE_PLUGINS_PATH}/available/$(PLUGIN_NAME) ${PLUGINS_PATH}/available; \
-		find /var/lib/dokku/ -xtype l -delete;\
+		find ${DOKKU_LIB_ROOT} -xtype l -delete;\
```

Comment:
This might need a trailing slash :)

Reply:
Done! Although I think `find` does recurse into the directory automatically.